docs/spelling_wordlist.txt

```diff
@@ -1594,24 +1577,41 @@ winrm
 workgroup
 workgroups
 wsman
+WTF
 wtf
 wtforms
 www
+XCom
+Xcom
 xcom
+XComArg
 xcomarg
+XComArgs
 xcomresult
+XComs
+Xero
+Xiaodong
 xml
 xpath
 xyz
 yaml
+Yamllint
+Yandex
 yandex
 yandexcloud
 yarnpkg
+Yieldr
 yml
 youtrack
 youtube
 zA
+Zego
+Zendesk
 zendesk
```

Comment:
IIRC the spell checker is case insensitive so we should be able to get rid of these duplicates?

Reply:
Actually this wordlist is used for both Sphinx spellchecker and the codespell and I think the Sphinx isn't case sensitive (and It's good as we can detect some casing that is wrong